tests/openadapt/test_summary.py

```diff
@@ -1,36 +1,41 @@
-"""
-Tests the summarization function in summary.py
-"""
-from fuzzywuzzy import fuzz
+"""Tests the summarization function in summary.py."""
 
+from fuzzywuzzy import fuzz
```

Comment:
I think we've been putting a new line between external library imports and ours

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507